amundsen-kube-helm/templates/helm/neo4j/values.yaml

```diff
@@ -19,7 +19,13 @@ config:
     heap_max_size: 23000m
     pagecache_size: 26600m
 neo4j:
-    backup:
-      enabled: false
-      s3Path: "s3://my-bucket/my-prefix"
-      schedule: "0 * * * *"
+  persistence: {}
+#  persistence:
+#    storageClass: gp2
```

Comment:
sorry for the n00b qq, but do we still need these lines after comment out?

Reply:
Hey @feng-tao don't worry about the n00b qq, it's still somewhat new to me to :)

Basically, the practice of providing an example values file seems to be standard. I know it appears funny though.

Take a look at the example airflow values file [here.](https://github.com/helm/charts/blob/master/stable/airflow/values.yaml)